docs/testing/regression-checklist/mapping.json

```
@@ -1,10 +1,10 @@
 [
 	{
-		"regex": "packages/connection/*",
+		"regex": "packages/connection/(?!CHANGELOG|changelog/)",
```

While we're at it, let's add the `projects/` prefix that we have since the big monorepo reorg. That'll prevent it from also matching PRs touching `projects/js-packages/connection/`
```suggestion
		"regex": "projects/packages/connection/(?!CHANGELOG|changelog/)",
```
For that matter, I wonder if we can anchor it at the start? Like
```suggestion
		"regex": "^projects/packages/connection/(?!CHANGELOG|changelog/)",
```
Although I don't know exactly what paths the app in question might pass.